<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Space Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="convTypeCB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="typeLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type of convertion:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstCoordLE">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="firstCoordLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R:</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondCoordLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G:</string>
    </property>
   </widget>
   <widget class="QLabel" name="thirdCoordLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>155</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondCoordLE">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="thirdCoordLE">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="inputLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input coordinates:</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Get result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondResultLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="thirdResultLE">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="thirdResultLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>315</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstResultLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>255</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>L:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondResultLE">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstResultLE">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>168</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
